docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+
+The **Google Drive connector** exposes Google Drive API response as a local variable called response.
+The following fields are available in the response variable:
+`googleDriveResourceId` - ID of a newly created resource
+`googleDriveResourceUrl` - human-readable URL of a newly created resource
+
+You can use an Output Mapping to map the response:
+
+1. Use **Result Variable** to store the response in a process variable. For example, `myResultVariable`.
+2. Use **Result Expression** to map fields from the response into process variables. For example:
+
+```
+{
+  "googleDriveResourceId":"1KMG4S_18kLNZa7ZCARtGIh8DmIoXXXXXXXXX",
+  "googleDriveResourceUrl":"https://docs.google.com/document/d/1KMG4S_18kLNZa7ZCARtGIh8DmIoXXXXXXXXX"
+}
+```
+
+# Appendix & FAQ
+
+## How can I get Google Drive OAuth token?
+
+Refer to the official Google Drive API documentation page.
+
+## I wish to do an ad-hoc test of the Google Drive connector without setting up complex OAuth flows. How can I do that?
+
+While this is highly not recommended due to security concerns, in certain cases, such as learning, exploring possibilities, or preparing a demo, you can still obtain a Google API token. For that, you will need to create a service account, and generate and download its keys. After that, you can execute this python script which will produce your OAuth token.
+
+```
+import google.auth
+import google.auth.transport.requests
+from google.oauth2 import service_account
+
+SCOPES = ['https://www.googleapis.com/auth/drive', 'https://www.googleapis.com/auth/drive.file', 'https://www.googleapis.com/auth/drive.appdata']
+SERVICE_ACCOUNT_FILE = 'google-service-account-creds.json'
+
+credentials = service_account.Credentials.from_service_account_file(SERVICE_ACCOUNT_FILE, scopes=SCOPES)
+auth_req = google.auth.transport.requests.Request()
+credentials.refresh(auth_req)
+print(credentials.token)
+```
+
+## Where do I get a parent folder ID?
+
+It is in the URL.
+
+![Google Drive connector get parent folder ID](../img/connectors-googledrive-get-parent-folder-id.png)
+
+## How do I set Additional properties or metadata?
+
+You can set any property from the Google Drive Create API. For example:
+
+```
+= {
+  "description":"myDescription"
+}
+```
+
+The unknown or mistyped properties will be ignored.
+
+## What are the limitations of the Additional properties or metadata?
+
+Some properties are applicable only for the token owners, like `folderColorRgb` and `starred`.
+
+## Where do I get Template ID?
+
+It is in the URL.
+
+![Google Drive connector get template ID](../img/connectors-googledrive-get-template-id.png)
+
+## Can you show me an example of a valid template?
+
+The valid template looks like the one on a screenshot.
+
+![Google Drive connector document template example](../img/connectors-googledrive-template-example.png)
+
+Now, in the Template variables field we can apply the following FEEL JSON object that has to be compatible with the Google Docs Requests API:
```

Comment:
Is "Template variables" field a UI element? If so, it should be bolded.

Reply:
Done